keras_contrib/layers/advanced_activations.py

```diff
@@ -283,3 +283,108 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 get_custom_objects().update({'Swish': Swish})
+
+
+class ReLUs(Layer):
+    """Rectified Linear Unit with Sigmoid to generate oscilations.
+
+    It allows an oscilation in the gradients when the weights are negative.
+    The oscilation can be controlled with a parameter, which makes it be close
+    or equal to zero. So, not all neurons are deactivated and it allows differentiability
+    in more parts of the function.
+
+    # Input shape
+        Arbitrary. Use the keyword argument `input_shape`
+        (tuple of integers, does not include the samples axis)
+        when using this layer as the first layer in a model.
+
+    # Output shape
+        Same shape as the input.
+
+    # Usage
```

Comment:
Can you put the examples at the end following this syntax:
https://github.com/keras-team/keras/blob/master/keras/layers/convolutional.py#L2245-L2256
One example should be enough.

Reply:
Sure, will do that.